change/@ni-nimble-blazor-92675eea-194d-4c8d-81ea-ff1049cd4f6c.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Reviewed the build scripts and package changes. Relying on blazor owners for C# proj, sln, nuget, etc changes